README.md

```diff
@@ -17,6 +17,7 @@ Gollum is a simple wiki system built on top of Git. A Gollum Wiki is simply a gi
 	* May be written in a variety of [markups](#markups).
 	* Can be edited with your favourite system editor or IDE (changes will be visible after committing) or with the built-in web interface.
 	* Can be displayed in all versions, reverted, etc.
+* Gollum strives to be compatible with GitHub wikis (see `--hyphened_tag_lookup`)
```

Comment:
Ok to mention github here, or do we have an alternative?

Reply:
Seems fair, since it is one of the original selling points of gollum!